torchreid/engine/image/am_softmax.py

```diff
@@ -247,10 +239,16 @@ def forward_backward(self, data):
                         continue
 
                     with torch.no_grad():
-                        trg_probs = torch.softmax(torch.stack(out_logits[trg_id]), dim=2).mean(dim=0)
+                        if self.loss_name in ['asl', 'am_asl']:
```

Comment:
Is this branch is supposed to be moved to the corresponding engine?

Reply:
I remember deleting it... Okay, I will check one more time if the  ASL loss is still present in the AM-Softmax engine